packages/office-addin-project/src/convert.ts

```diff
@@ -68,6 +70,10 @@ async function backupProject(backupPath: string) {
   });
 }
 
+export function convertManifest(manifestPath: string, outputPath: string) {
```

Comment:
Does this need to be exported?

Reply:
The idea is the customer could call this function directly and not use the backup and `package.json` check functions

Reply:
When would that be necessary?  The more pieces of the puzzle we expose the more we have to support customers usage of them and that limits how we change things in the future.  The goal is to get customer project from a xml based manifest project to a json base manifest project and we don't need to allow all the intermediate steps to be done separately.

Even if we did want to do this (and I'm saying we don't) this would go in the manifest package and not the project package.